src/User32.Desktop/User32.cs

```diff
@@ -483,6 +783,44 @@ public static partial class User32
         public static extern IntPtr GetSystemMenu(IntPtr hWnd, [MarshalAs(UnmanagedType.Bool)] bool bRevert);
 
         /// <summary>
+        /// Retrieves a handle to the Shell's desktop window.
+        /// <para>
+        /// Go to https://msdn.microsoft.com/en-us/library/windows/desktop/ms633512%28v=vs.85%29.aspx
+        /// for more information
+        /// </para>
+        /// </summary>
+        /// <returns>
+        /// C++ ( Type: HWND ) <br/> The return value is the handle of the Shell's desktop window. If
+        /// no Shell process is present, the return value is NULL.
+        /// </returns>
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern IntPtr GetShellWindow();
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern IntPtr LoadCursor(IntPtr hInstance, IntPtr lpCursorName);
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern IntPtr LoadIcon(IntPtr hInstance, string lpIconName);
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern IntPtr LoadIcon(IntPtr hInstance, IntPtr lpIconName);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern IntPtr RealChildWindowFromPoint(IntPtr hwndParent, POINT ptParentClientCoords);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool ScreenToClient(IntPtr hWnd, ref POINT lpPoint);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool SetWindowDisplayAffinity(IntPtr hWnd, int dwAffinity);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern unsafe uint RealGetWindowClass(
+            IntPtr hwnd,
+            [Friendly(FriendlyFlags.Array | FriendlyFlags.Out)] StringBuilder pszType,
```

Comment:
I think `char*` was what I meant. I was using it internally with `StringBuilder` and probably stopped the refactoring of this function on the middle. Sorry.